code/modules/crafting/smeltery.dm

```diff
@@ -0,0 +1,46 @@
+/*Crafting recipes for stuff made out of the blacksmtihing tools*/
+
+/datum/crafting_recipe/broadsword
+	name = "Smith Broadsword"
```

Comment:
You could say a forged blade (as long as people don't think the other meaning for forged being made fake) but making the code consistent to say smith would not be a huge undertaking.